src/data_stack/dataset/splitter.py

```diff
@@ -84,6 +89,42 @@ def get_indices(self, dataset_iterator: DatasetIteratorIF) -> List[List[int]]:
         return splits_indices
 
 
+class StratifiedSplitterImpl(SplitterIF):
+
+    def __init__(self, ratios: List[float]):
```

Comment:
What about adding a `shuffle_seed: int = None` parameter to the constructor, to shuffle the split indices?
Otherwise, we always take the same samples in the same order.   

The `train_test_split` function can take a shuffle and a random state parameter:  https://scikit-learn.org/stable/modules/generated/sklearn.model_selection.train_test_split.html

Reply:
Sure, we can pass a seed parameter. train_test_split does the shuffling anyways if stratify=True, but seeding is not such a bad idea I guess :D